cachito/web/api_v1.py

```diff
@@ -294,6 +294,8 @@ def create_request():
         )
         error = "Failed to schedule the task to the workers. Please try again."
         request.add_state("failed", error)
+        db.session.add(request)
```

Comment:
Do we really need to add the request again here? Note that the add_state call will call db.session.add for the state.

Reply:
Oh right, I had db.session.commit on wrong place before and it didn't work without this call. Removing that.